Documentation/quickstart/debugging.rst

```diff
@@ -49,63 +60,33 @@ Changing Debug Settings Quickly
 You can use the ``kconfig-tweak`` script that comes with the ``kconfig-frontends`` tools to quickly change debug settings,
 for instance turning them on or off before doing a build:
 
-.. code-block:: bash
+.. code-block:: console
 
    $ kconfig-tweak --disable CONFIG_DEBUG_NET
+   $ make oldconfig  # needed to have the kconfig system check the config
```

Comment:
I find olddefconfig better as it will without prompting set new symbols to their default value.

Reply:
@Ouss4 Cool, thanks. I accepted your suggestions.